include/wlr/types/wlr_pointer_gestures_v1.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_POINTER_GESTURES_V1_H
+#define WLR_TYPES_WLR_POINTER_GESTURES_V1_H
+
+#include <wayland-server.h>
+#include <wlr/types/wlr_pointer.h>
+
+struct wlr_pointer_gestures_v1 {
+	struct wl_global *wl_global;
+	struct wl_list resources;
+	struct wl_list swipes; // wlr_pointer_gesture_swipe_v1::link
```

Comment:
Could these two lists just be lists of `wl_resource`?

Reply:
Well, I don't really need the `wlr_pointer_gesture_swipe_v1` objects at all, would it be okay to just get rid of them?

Reply:
+1

Reply:
done